applications/debug/unit_tests/nfc/nfc_test.c

```diff
@@ -466,6 +466,10 @@ static void mf_classic_generator_test(uint8_t uid_len, MfClassicType type) {
     nfc_device_free(nfc_keys);
 }
 
+MU_TEST(mf_mini_file_test) {
```

Comment:
Add 
MU_RUN_TEST(mf_mini_file_test);
in MU_TEST_SUITE(nfc) please

Reply:
Missed that, thanks!